site/specs/webrtc.yml

```diff
@@ -2603,13 +2911,39 @@ components:
       schema:
         type: string
       example: 62e0ecb9-0b1b-5115-aae4-4f36123d6bb1
+    dialogId:
+      name: dialogId
+      in: path
+      description: The ID of the Dialog assigned to the Participant
+      required: true
+      schema:
+        type: string
+      example: 62e0ecb9-0b1b-5115-aae4-4f36123d6bb1
   requestBodies:
+    createEventRequest:
+      description: Create an event-notification request body.
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/eventBody'
     createParticipantRequest:
       description: Create participant request body.
       content:
         application/json:
           schema:
             $ref: '#/components/schemas/createParticipantRequest'
+    createDialogRequest:
+      description: Body for Dialog creation
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/createDialogRequest'
+    updateDialogRequest:
+      description: Body for Dialog creation
```

Comment:
This description should reference update and not creation

Reply:
done - thanks